webpack/assets/javascripts/react_app/components/notifications/index.js

```diff
@@ -37,7 +36,7 @@ class notificationContainer extends React.Component {
     if (error) {
       const { response: { status } = {} } = error;
       if (status === 401) {
-        redirectToLogin();
+        window.location.reload(true);
```

Comment:
reloading the entire page works perfectly fine for me.

Reply:
getting `401` error, in this case, means that the user session is expired, why not redirect to login page?

Reply:
@amirfefer hello :) yes, user session is expired but the login page in the case of `external` users is/might be different, therefore we must leave it to the server to decide where it needs to redirect.
In case of an `internal` user it will still redirect to the foreman login page.

Let me know if this answers your question, if not please let me know your thoughts :)

Reply:
thanks :+1:

Reply:
should we reuse `reloadPage` function from foreman_navigation (e.g. in case we change the implementation in the future)?